SwiftUI/ReviewWQuestion-Calculator/ContentView.swift

```diff
@@ -4,11 +4,360 @@
 //
 
 import SwiftUI
+import Foundation
+
+enum Buttons : String{
+    case zero = "0"
+    case one = "1"
+    case two = "2"
+    case three = "3"
+    case four = "4"
+    case five = "5"
+    case six = "6"
+    case seven = "7"
+    case eight = "8"
+    case nine = "9"
+    case plus = "+"
+    case minus = "ㅡ"
+    case equal = "="
+    case divide = "/"
+    case multiple = "X"
+    case dot = "."
+    case percentage = "%"
+    case clear = "AC"
+    case opposite = "+/-"
+    
+    var bgColor : Color {
+        switch self{
```

Comment:
```suggestion
        switch self {
```

중괄호 앞에 띄어쓰기를 하지 않는 이유가 있으실까요?

Reply:
주의하도록 하겠습니다..